internal/pkg/rng/tpm.go

```
@@ -22,8 +21,8 @@ import (
 func TPMSeed() error {
 	t, err := tpm.Open()
 	if err != nil {
-		// if the TPM is not available or not a TPM 2.0, we can skip the PCR extension
```

this error is now not returned anymore by the library